ivy/utils/decorator_utils.py

```diff
@@ -183,8 +183,6 @@ def handle_get_item(fn):
     def wrapper(inp, query, **kwargs):
         try:
             res = inp.__getitem__(query)
-        except IndexError:
-            raise
```

Comment:
need to remove this so the backend implementation can be called, which handles an IndexError when the query is a multi-dimensional tensor